packages/hash/frontend/src/components/layout/PageSidebar/WorkspaceSwitcher.tsx

```diff
@@ -18,38 +34,68 @@ const truncateText = (text: string) => {
 };
 
 export const WorkspaceSwitcher: VFC<WorkspaceSwitcherProps> = () => {
```

Comment:
Inclined to agree on this. @MaggieAppleton for thoughts.

It would be good to still expose the full name of the current org in the event it has been truncated - perhaps via alt-text or a tooltip that appears on prolonged hover? Happy to consider that OOS of this PR.

Reply:
Currently active workspace should still show up in the dropdown menu as we list other metadata there (namely, number of workspace users). The currently active workspace should be highlighted with a blue background – eg. our universal design pattern for actively selected items.

Reply:
Thanks for clarifying @MaggieAppleton , resolving this